Cargo.lock

```diff
@@ -3764,9 +3764,9 @@ checksum = "830d08ce1d1d941e6b30645f1a0eb5643013d835ce3779a5fc208261dbe10f55"
 
 [[package]]
 name = "libc"
-version = "0.2.169"
+version = "0.2.170"
```

Comment:
Bumped due to `solana-logger` update and its dependency requirements: https://crates.io/crates/solana-logger/2.3.1/dependencies